### PR TITLE
Create contributions for PR reviews

### PR DESCRIPTION
**Pull request by galfert**
Refs #49

This should create the actual Contribution transactions for the PR reviews.

I could not test this myself, because I don't have a local devchain running.

### CODE REVIEW

File: scripts/review-kredits.js [hunk 1]
```diff
@@ -121,6 +136,17 @@ async function initializeKredits () {
   return kredits;
 }
 
+function createContribution(contributorName, contributionAttributes, Contribution) {
+  console.log(`Creating contribution token for ${contributionAttributes.amount}₭S to ${contributorName} for ${contributionAttributes.description}...`);
+
+  return Contribution.add(contributionAttributes).catch(error => {
```
Comment (bumi) [inline scripts/review-kredits.js:142]: why do you pass in `Contribution` here and not use `kredits.Contribution` from the global variable?

**Review by bumi — APPROVED** [review]
looks good to me! 
I still have local issues running the aragon dependency here, but it looks good to me. and I guess with the reviews we have to test that on live anyway. 
👍

File: scripts/review-kredits.js (hunk 1 above)
Comment (galfert) [reply scripts/review-kredits.js:142]: `kredits` is not global.

Comment (bumi) [reply scripts/review-kredits.js:142]: oh true. 
though I guess it could be?

**Pull request merged**